analyzer/src/main/kotlin/managers/Sbt.kt

```diff
@@ -63,11 +63,14 @@ class Sbt(
         // https://www.scala-sbt.org/1.x/docs/Command-Line-Reference.html#Command+Line+Options.
         private val CI_MODE = "-Dsbt.ci=true".addQuotesOnWindows()
 
+        // Disable console colors explicitly since at some cases CI_MODE is not enough.
```

Comment:
s/since/as/
s/at/in/

Reply:
And sorry for not noticing this before, but the commit message body text is not properly hard-wrapped at column 75.

Reply:
Fixed